nixos/modules/services/backup/postgresql-backup.nix

```
@@ -160,6 +167,14 @@ in
             assertion = cfg.backupAll -> cfg.databases == [ ];
             message = "config.services.postgresqlBackup.backupAll cannot be used together with config.services.postgresqlBackup.databases";
           }
+          {
+            assertion = cfg.backupAllExcept != [ ] -> cfg.databases != [ ];
+            message = "services.postgresqlBackup.backupAllExcept cannot be used together with services.postgresqlBackup.databases";
```

This assertion smells.

Shouldn't this be


```suggestion
            assertion = cfg.backupAllExcept != [ ] -> cfg.databases == [ ];
            message = "services.postgresqlBackup.backupAllExcept cannot be used together with services.postgresqlBackup.databases";
```

But wouldn't that fail as we set cfg.databases below?

---

yep, I've removed the assertion.